plugins/modules/storage/zfs/zfs.py

```diff
@@ -37,6 +37,11 @@
       - A dictionary of zfs properties to be set.
       - See the zfs(8) man page for more information.
     type: dict
+notes:
+  - C(check_mode) is supported, but in certain situations it may report a task
+    as changed that would not reported as changed if C(check_mode) were disabled.
+    For example, this might occur when the zpool C(altroot) option is set or when
+    a size is written using human-readable notation such as C(1024M) or C(2G).
```

Comment:
I'm not sure about this, but it might be good to continue the sentence like "... instead of using bytes" or something like that, as it's not clear enough right now what is the alternative IMHO.

Reply:
Good point. I added some more explanation. I also changed the units so that they all represent the same quantity.